src/open_r1/grpo.py

```diff
@@ -171,7 +164,7 @@ def main(script_args, training_args, model_args):
     def make_conversation(example):
         return {
             "prompt": [
-                {"role": "system", "content": SYSTEM_PROMPT},
+                {"role": "system", "content": training_args.system_prompt},
```

Comment:
caution, this can be None.
```python
prompt = []

if training_args.system_prompt is not None:
    prompt.append({"role": "system", "content": training_args.system_prompt})

prompt.append({"role": "user", "content": example["problem"]})
```